lib/styles/fonts.css

```diff
@@ -1,10 +1,4 @@
-/*
-Include the following in your HTML head:
-
-<style>
-  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
-</style>
-*/
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
```

Comment:
nit: I'd like a font solution that doesn't depend on additional network calls. Maybe worth filing an issue for later? I'm guessing all these css files and woff files could be dropped into assets without too much work.

Reply:
Created an issue https://github.com/PERTS/perts-ui/issues/13 to address this. Thanks.